testOutputVcf.py

```diff
@@ -0,0 +1,294 @@
+
```

Comment:
This script needs a negative control to make sure it fails when it's supposed to.... I don't think this testing code need to be in this repo, I'll put it in a separate repo to reduce complications.